CMakeLists.txt

```diff
@@ -7,17 +7,17 @@ cmake_policy(VERSION 3.0)
 
 project(PonyProject CXX)
 
-option(PONY_BUILD_DOCS "Build Pony project's docs" OFF)
+#option(PONY_BUILD_DOCS "Build Pony project's docs" OFF)
 option(PONY_BUILD_TESTS "Build Pony project's tests" ON)
 
-if (UNIX)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Werror -pedantic")
-    set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -s -O3")
-    set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -g3 -ggdb3 -O0")
-elseif(WIN32)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /std:c++14 /MP /W3")
-    add_definitions(-D_CRT_SECURE_NO_WARNINGS)
-endif()
+#if (UNIX)
```

Comment:
Commented code. If you are not going to use it, please delete it.

Reply:
ok